radio/src/translations/de.h

```diff
@@ -734,7 +734,7 @@
 #define TR_V1                          "V1"
 #define TR_V2                          "V2"
 #define TR_DURATION                    "Dauer"
-#define TR_DELAY                       "Verzög."
+#define TR_DELAY                       "Verzögerung"
```

Comment:
The abbreviation here was probably selected with a reason. The B&W LCD radios have much less space and sometime longer texts may not fit.

Reply:
![image](https://user-images.githubusercontent.com/15068446/192163073-69ac9975-4079-4482-bc30-9922c8491d9f.png)
Are there any B&W radios with smaller display?

Reply:
no, the smalles ist 128x64 pixel, and that is what the QX7 should have, when I read to code correctly